drivers/sensor/icp101xx/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources(icp101xx.c Devices/Drivers/ICP101xx/Icp101xx.c)
+zephyr_library_include_directories(. Devices/Drivers/ICP101xx/ EmbUtils)
```

Comment:
strange in github page, I see a space between EmbUtils and Devices.
This is a list of 3 paths ?:
.
Devices/Drivers/ICP101xx/
EmbUtils

maybe better to put 
./
./Devices/Drivers/ICP101xx/
./EmbUtils

Reply:
Done